pommerman/agents/base_agent.py

```diff
@@ -1,8 +1,10 @@
+from .. import characters
+
 
 class BaseAgent:
```

Comment:
With the change to characters.Bomber and the push to make it default everywhere, we should additionally change it from self._agent to self._character.